pyproject.toml

```diff
@@ -15,22 +15,28 @@ exclude = ["src/book/_build"]
 [tool.poetry.dependencies]
 python = ">=3.9,<=3.12"
 pyyaml = "^6.0.1"
-
+gitpython = "^3.1.43"
 
 [tool.poetry.group.book.dependencies]
 jupyter-book = "^1.0.0"
 docutils = "0.17.1"
 sphinxcontrib-bibtex = "^2.6.2"
+pandas = "^2.2.2"
+pyarrow = "^16.0.0"
+jupyterlab = "^4.2.3"
 
 [tool.poetry.group.dev.dependencies]
```

Comment:
I noticed that `matplotlib` and `seaborn` might be requirements in the notebook - consider adding these or other dependencies here.